mslice/app/qapp.py

```diff
@@ -0,0 +1,20 @@
+from mslice.util.qt.QtWidgets import QApplication
+
+qApp = None
+
+
+def create_qapp():
+    """
+    Create a qapplication for non gui plots
+    """
+    global qApp
+
+    if qApp is None:
```

Comment:
It would be safer if we have
```python
if qApp is None:
    instance = QApplication.instance() 
    if instance is None:
        instance = QApplication['mslice']
        instance.lastWindowClosed.connect(instance.quit)
    qApp = instance

return qApp
```
as it could be possible that `qApp` is `None` but the `QApplication` object has been created by something else.